modules/@apostrophecms/schema/ui/apos/components/AposInputRelationship.vue

```diff
@@ -21,14 +21,14 @@
             :label="browseLabel"
             :modifiers="['small']"
             type="input"
-            @click="choosing=true"
+            @click="choose"
           />
         </div>
         <AposSlatList
           class="apos-input-relationship__items"
           v-if="next.length"
           @input="updateSelected"
-          @item-clicked="openRelationshipEditor"
+          @item-clicked="editRelationship"
```

Comment:
It looks like we have a regression with this no longer working. editing the relationship, that is.